.github/workflows/new-release-tag-docker.yml

```diff
@@ -47,6 +63,34 @@ jobs:
             Fill in with release drafter information manually for now, then publish.
           draft: true
           prerelease: false
+      - name: Upload Release Asset
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./build/defectdojo-${{ steps.package-helm-chart.outputs.chart_version }}.tgz
+          asset_name: defectdojo-${{ steps.package-helm-chart.outputs.chart_version }}.tgz
+          asset_content_type: application/octet-stream
+      - name: Update Helm repository index
```

Comment:
I just learned about https://github.com/actions/upload-release-asset, so it might actually be better to also use it to upload it. The example workflow should nearly be a copy-paste to replace this step, and maybe a slight adjustment to the asset content type.

```      
      - name: Upload Release Asset
        id: upload-release-asset 
        uses: actions/upload-release-asset@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          upload_url: ${{ steps.create_release.outputs.upload_url }}
          asset_path: ./build/defectdojo-${{ steps.package-helm-chart.outputs.chart_version }}.tgz
          asset_name: defectdojo-${{ steps.package-helm-chart.outputs.chart_version }}.tgz
          asset_content_type: application/tar+gzip
```

Reply:
Sorry, I don't understand the other change you are requesting. `actions/upload-release-asset@v1` is already used to upload the chart tarball.

Reply:
Nevermind, I got mixed up, sorry about that!